File: my-folder/1013-fibonacci-number/solution.cpp
class Solution {
public:
    int fib(int n) {
        int n_2 = 0;
        int n_1 = 1;
        int fib = 0;
        if(n<2) return n;
        
       else{
            for(int i = 1; i < n; i++){
            fib = n_2 + n_1;
           n_2 = n_1;
           n_1 = fib;
           }
       }
        return fib;

    }
};

File: my-folder/1013-fibonacci-number/solution.py
class Solution:
    def fib(self, n: int) -> int:
        one,two, fib = 0, 1, 0
        if n==0 or n==1:
            return n
        for i in range(2, n+1):
            fib = one + two
            one = two
            two = fib
        return fib 

